digdag-standards/src/main/java/io/digdag/standards/operator/td/TDOperator.java

```diff
@@ -67,7 +67,13 @@ public static TDOperator fromConfig(BaseTDClientFactory clientFactory, SystemDef
 
         TDClient client = clientFactory.createClient(systemDefaultConfig, env, config, secrets);
 
-        return new TDOperator(client, database);
+        return new TDOperator(client, database, secrets);
+    }
+
+    public void updateApikey(SecretProvider secrets)
+    {
+        String apikey = TDClientFactory.getApikey(secrets);
+        client = client.withApiKey(apikey);
     }
 
     static final RetryExecutor defaultRetryExecutor = retryExecutor()
```

Comment:
This defaultRetryExecutor is still used in TDJobOperator. Is this OK?

Reply:
Thanks, I fixed TDJobOperator also here.
https://github.com/treasure-data/digdag/pull/1362/commits/87cf2ba7c7b3232be7228e896d2e0892a03094cf